File: AlgoData/PrepLab/Assignment2/2.c
#include <stdio.h>

int main(int argc, const char* argv[] ){
    FILE *fp = fopen("/Users/Daniel/Documents/AlgoData/PrepLab/Assignment2/2input.txt", "r" );
    if(fp == NULL){
        printf("Error reading file");
        return 0;
    }

    int c;
    do{
        c = fgetc(fp);
        if(c == 'a'){
            printf("%c", 'x');
        } else{
            printf("%c",c);
        }
    } while (c != EOF);

    system("pause");
    return 0;
}

File: AlgoData/PrepLab/Assignment2/2.java
import java.io.*;

public class 2{

    public static void main(String[] args) throws IOException{
        FileReader fr;
        fr=new FileReader("/Users/Daniel/Documents/AlgoData/PrepLab/Assignment2/2input.txt");
        BufferedReader br = new BufferedReader(fr);
        int c;
        while((c = br.read()) != -1){
            if((char)c == 'a'){
                System.out.println('x');
            } else{
                System.out.println((char)c);
            }
        }
    }
}